sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/models/Page.java

```diff
@@ -0,0 +1,37 @@
+package com.microsoft.appcenter.storage.models;
+
+import java.util.List;
+
+public class Page<T> {
+
+    /**
+     * Documents in the page.
+     */
+    private List<Document<T>> documents;
+
+    private DocumentError documentError;
```

Comment:
The getter is called `getError` and the backing field is `documentError`, rename field to error?